syntax = "proto3";
package helmsley;
message Request{
    int32 client_id = 1;
    string req_msg = 2;
}

message commonResponse{
    bool success = 1;
    string res_msg = 2;
}

message scoreInfo{
    int32 rgroup_id = 1;
    int32 rank_id = 2;
    float rank_score = 3;
    repeated float raw_score = 4;
    repeated float vol_score = 5;
}